lldb/source/Symbol/SwiftASTContext.cpp

```diff
@@ -5431,7 +5431,9 @@ static CompilerType BindAllArchetypes(CompilerType type,
     return type;
   ExecutionContext exe_ctx;
   exe_scope->CalculateExecutionContext(exe_ctx);
-  return runtime->DoArchetypeBindingForType(*frame, type);
+  if (auto bound = runtime->DoArchetypeBindingForType(*frame, type))
+    return bound;
+  return type;
```

Comment:
I thought we should really never fail unless we have a type parameter we're trying to bind.
( i.e. https://github.com/apple/llvm-project/pull/549 )

Reply:
I take this back.